### PR TITLE
fix: change in_active to more clear words

### PR DESCRIPTION
**Pull request by muniter**
This won't break compatibility, changes `in_active` for `inactive` and `show_in_inactive` for `show_while_inactive` and will make the documentation more clear.

I tested locally with the samples, if you like this approach test with your setup that perhaps is more complex.

### CODE REVIEW

File: lua/windline/init.lua [hunk 1]
```diff
@@ -198,6 +201,10 @@ end
 
 
 M.add_status = function(lines)
+    for _, line in ipairs(lines) do
+        line.inactive = line.in_active
+        line.show_while_active = line.show_in_active
+    end
```
Comment (muniter) [inline lua/windline/init.lua:207]: By just aliasing we avoid the confusion with the old names, performance wise is only while setting up and it's probably not measurable.

**Comment by windwp** [comment]
I think maybe just  use `show_active` and we description mean it always show active status.

File: lua/windline/init.lua (hunk 1 above)
Comment (muniter) [reply lua/windline/init.lua]: This was wrong, if `line.inactive` was nil then it would break a status line using the new table keys.

**Comment by muniter** [comment]
@windwp I have renamed to `always_active`

**Comment by windwp** [comment]
Hi, i add a message to tell user update change so we can remove that code later
if you doesn't have any change to commit i will merge.

**Comment by muniter** [comment]
@windwp I don't have any more changes! LGTM

File: lua/windline/init.lua [hunk 2]
```diff
@@ -198,8 +198,13 @@ end
 
 
 M.add_status = function(lines)
+    -- FIXME update change. It will be remove on the future
     for _, line in ipairs(lines) do
+        utils.update_check(line.in_active ~= nil,
+           "You need change 'in_active' to 'inactive'. ':%s/in_active/inactive/g'")
         line.inactive = line.inactive or line.in_active
+        utils.update_check(line.show_in_active ~= nil,
+        "You need change 'show_in_active' to 'always_active'. ':%s/show_in_active/always_active/g'")
```
Comment (muniter) [inline lua/windline/init.lua:207]:
```suggestion
        "You need to change status line 'show_in_active' to 'always_active'. ':%s/show_in_active/always_active/g'")
```

File: lua/windline/init.lua [hunk 3]
```diff
@@ -198,8 +198,13 @@ end
 
 
 M.add_status = function(lines)
+    -- FIXME update change. It will be remove on the future
     for _, line in ipairs(lines) do
+        utils.update_check(line.in_active ~= nil,
+           "You need change 'in_active' to 'inactive'. ':%s/in_active/inactive/g'")
```
Comment (muniter) [inline lua/windline/init.lua:204]:
```suggestion
           "You need to change status line 'in_active' to 'inactive'. ':%s/in_active/inactive/g'")
```

**Comment by windwp** [comment]
thank

**Pull request merged**